docs/okeps/okep-5217-routed-ingress-l2-udn-without-ipam.md

```diff
@@ -0,0 +1,116 @@
+# OKEP-5217: Routed ingress support for L2 UDN without dynamic IP allocation
+
+* Issue: [#5217](https://github.com/ovn-org/ovn-kubernetes/issues/5217)
+
+## Problem Statement
+
+There is an increased interest for VMs in Kubernetes, which is generating the
+desire to import VMs from another virtualization platforms.
+
+VMs running in virtualization platforms are usually using static IP allocation,
+where IPs are assigned directly in the guest.
+
+One of the features most sought after by virtualization users is routed ingress
+for L2 UDN, something which - right now - OVN-Kubernetes allows using the BGP
+feature.
+
+The current gap in OVN-Kubernetes is to provide routed ingress to VMs whose IPs
+were assigned statically in the guest. This enhancement plans to bridge that
+gap.
+
+## Goals
+
+- Detect/expose the MAC for the VM to be configured in OVNK for the VM (or disable port security)
```

Comment:
We should generalize this, "detect", "expose" or "port security" are implementation details, so we can decide later on the enhancement the alternatives

maybe something like 
```
Allow UDN traffic for virtujal machine guest internal mac addresses.
```